exporters/otlp/otlpmetric/client.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//go:build go1.18
+// +build go1.18
+
+package otlpmetric // import "go.opentelemetry.io/otel/exporters/otlp/otlpmetric"
+
+import (
+	"context"
+
+	mpb "go.opentelemetry.io/proto/otlp/metrics/v1"
+)
+
+// Client handles the transmission of OTLP data to an OTLP receiving endpoint.
+type Client interface {
```

Comment:
Why do we need to export this?

Reply:
This is how we differentiate between HTTP and GRPC.  Taken from the otlptrace.  We wouldn't be able to New(Client) an exporter without it.

Reply:
I was considering if both `Client` and `Exporter` are needed at all. If both of these types were moved into an internal package both the http and grpc exporters can still provide a `New` function that returns a `"go.opentelemetryi.io/sdk/metric".Exporter` by using these internal types.

The big issue I had with this is that it would diverge with the setup from the `otlptrace` package. I figured consistency was more important here, but I'm not 100% sure that is true.

Reply:
Keeping as is for consistency.